prestoadmin/server.py

```diff
@@ -125,6 +127,15 @@ def uninstall():
         print('Package uninstalled successfully on: ' + env.host)
 
 
+@task
+@requires_topology
+def upgrade(local_package_path, local_config_dir):
+    configure_cmds.gather_directory(local_config_dir)
```

Comment:
Per discussion, we're going to leave this because it will allow us to handle required configuration changes on upgrade in the future.